docker/README.md

```diff
@@ -64,8 +64,8 @@ bitcoin into.
 $ export NETWORK="simnet" 
 
 # Create persistent volumes for alice and bob.
```

Comment:
nit: we generally prefix commits with the affected package, as per [go guidelines](https://golang.org/doc/contribute.html#first_line). In this case there isn't strictly a package but `docker:` is probably more fitting :)

Reply:
np.  prefixed the commit with `docker`